src/h3lib/lib/geoCoord.c

```diff
@@ -294,35 +294,62 @@ double H3_EXPORT(edgeLengthM)(int res) {
     return lens[res];
 }
 
-/** @brief Number of unique valid H3Indexes at given resolution. */
+/**
+ * Number of cells (hexagons and pentagons) for a given resolution,
+ * which works out to be `2 + 120*7^r` for resolution `r`
+ *
+ * # Mathematical notes
+ *
+ * Let h(n) be the number of children n levels below
+ * a single *hexagon*.
+ *
+ * Then h(n) = 7^n.
+ *
+ * Let p(n) be the number of children n levels below
+ * a single *pentagon*.
+ *
+ * Then p(0) = 1, and p(1) = 6, since each pentagon
+ * has 5 hexagonal immediate children and 1 pentagonal
+ * immediate child.
+ *
+ * In general, we have the recurrence relation
+ *
+ * p(n) = 5*h(n-1) + p(n-1)
+ *      = 5*7^(n-1) + p(n-1).
+ *
+ * Working through the recurrence, we get that
+ *
+ * p(n) = 1 + 5*\sum_{k=1}^n 7^{k-1}
+ *      = 1 + 5*(7^n - 1)/6,
+ *
+ * using the closed form for a geometric series.
+ *
+ * Using the closed forms for h(n) and p(n), we can
+ * get a closed form for the total number of cells
+ * at resolution r:
+ *
+ * c(r) = 12*p(r) + 110*h(r)
```

Comment:
Minor nit, but can you surround the inline algebra with backticks and put three backticks above and below each block of algebra? That should improve the rendering of this when the site is regenerated.

PS: @isaacbrodsky you'll need to regen the site ;)

Reply:
I'm still unclear about which comment blocks actually feed into the docs. Shouldn't it be `h3api.h.in`, at least, eventually/ideally?

Reply:
Oh, then I am unclear as well. If they're only from the h3api.h file then you can ignore that and/or move this comment over there, because keeping it here where few will see it feels like a crime. ;)

Reply:
The website h3geo.org isbeing generated from the `docs/` folder, not from the C comments. The comments are only being used by Doxygen, and the output of that is not being hosted AFAIK. The files in `docs/` are regular Markdown files.

The downside is we need to manually duplicate a bunch of comments over to the Mardkwon file, the upside is we have more control over how the website is generated.